### ISSUE TITLE
Math not rendering properly

### ISSUE THREAD

**Issue by r-ford** [ProjectPythia/radiative-feedback-cookbook#15]
![image](https://github.com/ProjectPythia/radiative-feedback-cookbook/assets/50349951/d38a1064-44e8-4aea-8728-62ba449f6afe)

A lot of the equations in notebooks are getting cut off at the top. Is there a fix to this?

**Comment by brian-rose** [comment]
Can you link to particular sections? I'm not seeing that.

**Comment by r-ford** [comment]
I see it in my notebook and also the [EBM notebook](https://projectpythia.org/radiative-feedback-cookbook/notebooks/foundations/energy-balance-model.html). Could be a browser thing.

![image](https://github.com/ProjectPythia/radiative-feedback-cookbook/assets/50349951/af87cf14-46ed-48be-926b-2cd185a22290)

**Comment by brian-rose** [comment]
Yes, I don't see that in my view. Would be helpful to note here which browser you are using.

Have you noticed similar equation display issues in other cookbooks?

**Comment by tyfolino** [comment]
I see that too, but only for the very first equation. All others are ok. Using Opera browser.

![image](https://github.com/ProjectPythia/radiative-feedback-cookbook/assets/31661758/43510bc4-9f3d-461d-9db9-d5ec6d963e5e)

**Comment by r-ford** [comment]
I don't see it in [this section of the EOFs cookbook](https://projectpythia.org/eofs-cookbook/notebooks/eof-intro.html#the-method) at least. I'm seeing it on Firefox/Windows but not Firefox/Ubuntu.

**Comment by brian-rose** [comment]
Will be good to revisit this now that we are rendering the book with MyST. How's the math??

**Comment by r-ford** [comment]
Looks fine to me, so this probably won't be an issue anymore.
